scootapi/server/kill_job_test.go

```diff
@@ -0,0 +1,64 @@
+package server
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+
+	"github.com/scootdev/scoot/saga"
+	"github.com/scootdev/scoot/sched/scheduler"
+	"github.com/scootdev/scoot/scootapi/gen-go/scoot"
+)
+
+var mockCtrl *gomock.Controller
+
+/**
```

Comment:
nit: I think the extra * is unnecessary in these block comments. See https://golang.org/doc/effective_go.html#commentary